modules/delegates.js

```diff
@@ -220,26 +220,12 @@ __private.forge = function (cb) {
 		}
 
 		library.sequence.add(function (cb) {
-			async.series({
-				getPeers: function (seriesCb) {
-					return modules.transport.getPeers({limit: constants.maxPeers}, seriesCb);
```

Comment:
- Transport.prototype.getPeers ->
  - Broadcaster.prototype.getPeers ->
    - Peers.prototype.list
    - modules.peers.getConsensus

So now there is no need to call `modules.peers.getConsensus` inside `Broadcaster.prototype.getPeers`, that call should be removed.

Reply:
Also there is `self.consensus` variable in `logic/broadcaster` which looks like it not used at all.

Reply:
`Peers.prototype.list` description is invalid, that function is not calculating consensus and not returning it.

Reply:
You mean `Peers.prototype.getConsensus` - I agree, `calculateConsensus` would be a better fit.

Reply:
Broadcaster calculates consensus only for logging purposes now. If I remove the call before broadcasting it will not happen anymore. I agree that consensus variable in `broadcaster.getPeers` is local and shouldn't be assigned to `self` object anymore.

Reply:
I mean https://github.com/MaciejBaj/lisk/blob/6c20e0927a95950170bf177058c985008b2721eb/modules/peers.js#L456;L468

But yes, change `getConsensus` to `calculateConsensus` is nice idea also. :)